Frontend/CO.CDP.OrganisationApp/Pages/OneLogin.cshtml.cs

```diff
@@ -66,6 +66,12 @@ private async Task<IActionResult> UserInfo()
                 session.Set(Session.UserDetailsKey, ud);
             }
 
+            var personInviteId = session.Get<Guid?>("PersonInviteId");
+            if (personInviteId != null)
+            {
+                return RedirectToPage("ClaimOrganisationInvite", new { personInviteId });
```

Comment:
@dharmverma Based on your recommendation I am now redirecting to another page to handle the invitation claim.